installers/source/centos.sh

```diff
@@ -333,13 +333,18 @@ install_pdo_sqlsrv () {
 }
 
 install_oracle () {
-  yum install -y libaio systemtap-sdt-devel $DRIVERS_PATH/oracle-instantclient-*-21.*.rpm
+  CLIENT_VERSION=$(ls -f $DRIVERS_PATH/oracle-instantclient*-*-[12][19].*.0.0.0*.x86_64.rpm | grep -oP '([1-9]+)\.([1-9]+)' | head -n 1)
```

Comment:
@daniilly This line gives more than 2 combinations of `19` and `21`. Maybe we could specify exactly this 2 versions somehow? Not sure, whether `11` for example would lead to any error.

Reply:
@alexanderko, you are correct. However, we have specified in the guide which versions should be used. While Oracle Instant Client v11 is still downloadable, it is quite outdated, dating back to 2017.